app/src/contexts/UpdateCourseContext.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState, createContext, useContext, useMemo } from "react";
+
+// Do not remove default unused vars as typescript uses this for type hinting
+const defaultUpdateCourseContextState = {
+  updateCourse: {},
+  setUpdateCourse: (CourseDetails) => {},
+};
+const UpdateCourseContext = createContext(defaultUpdateCourseContextState);
+
+export function UpdateCourseContextProvider({ children }) {
+  const [updateCourse, setUpdateCourse] = useState(null);
```

Comment:
We can have better naming conventions for this because you don't set 'updateCourse'. But in light that this is the last sprint, we can ignore this and focus on other more important tasks.

Reply:
Understood. My bad